webapp/src/components/modals/channel_settings/select_channel_subscription.tsx

```diff
@@ -14,12 +14,14 @@ type Props = SharedProps & {
 type State = {
     error: string | null;
     showConfirmModal: boolean;
+    subscriptionToDelete: ChannelSubscription | null;
 }
 
 export default class SelectChannelSubscriptionInternal extends React.PureComponent<Props, State> {
     state = {
         error: null,
         showConfirmModal: false,
+        subscriptionToDelete: null,
     };
 
     handleDeactivateCancel = () => {
```

Comment:
```suggestion
    handleDeleteCancel = () => {
```

Reply:
I'm going to rename to `handleCancelDelete` to match the naming convention in `edit_channel_settings.tsx`